terroroftinytown/tracker/base.py

```diff
@@ -16,10 +16,11 @@ def get_current_user(self):
         if username_raw and token:
             username = username_raw.decode('ascii')
 
-            if username:
-                return User.check_account_session(username, token)
+            if username and User.check_account_session(username, token):
+                return username
 
     def prepare(self):
+        self.application.log_filter.username = self.get_current_user()
```

Comment:
Yeah, I'm not really happy with this implementation, but it seemed to work (although Travis says otherwise. :-) ). I don't (yet) understand what you are suggesting, though. I wanted to avoid changing the actual logging calls if I could avoid it.